client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -16,7 +16,7 @@ import SectionHeader from 'components/section-header';
 import SiteToolsLink from './link';
 import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
 import { isJetpackSite, getSiteAdminUrl } from 'state/sites/selectors';
-import { isSiteAutomatedTransfer, isVipSite } from 'state/selectors';
+import { isVipSite } from 'state/selectors';
```

Comment:
Is this removed on purpose? "Atomic" (Automated transfer) sites shouldn't be able to disconnect